warehouse/core/src/main/java/datawave/edge/protobuf/EdgeData.java

```diff
@@ -308,7 +308,7 @@ public EdgeValue getDefaultInstanceForType() {
             return defaultInstance;
         }
 
-        private final com.google.protobuf.UnknownFieldSet unknownFields;
+        private transient final com.google.protobuf.UnknownFieldSet unknownFields;
```

Comment:
The comment at the top says this was generated by protoc. It says to not edit, I do not know the impact if we do.

Reply:
This might be something where we're supposed to be generating this at build time instead of generating it once and adding it to source control by hand.

Reply:
missed that -- intellij collapsed the comment at the top.  reverted.